yevhenii.dolmatov/hash.rb

```diff
@@ -0,0 +1,21 @@
+hash = {
+  'Yes' => 23,
+   'b' => 'travel',
+   'yesterday' => 34,
+   5 => '234',
+   yesss: :fg,
+   try: 30,
+   key: 'some value',
+   ' yesterday1 ' => 34,
+   ' yesteryear ' => 2014 
+  }
+
+  counter = 0
```

Comment:
It would be better to avoid having an extra iterator variable `counter`

Method `Enumerable#count` allows passing a `counter = arr_of_hash_keys.hash.keys.count { |key| key.start_with?("yes") }`